tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -399,7 +399,15 @@ def validateRace():
     """Validate race."""
     return make_validator(
         lambda value: value >= 0 and value <= 2,
-        lambda value: f"{value} is not greater than or equal to 0 or smaller than or equal to 1.",
+        lambda value: f"{value} is not greater than or equal to 0 or smaller than or equal to 2.",
+    )
+
+
+def validateRptMonthYear():
+    """Validate RPT_MONTH_YEAR."""
+    return make_validator(
+        lambda value: value[2:8].isdigit(),
```

Comment:
is `[2:8]` always going to be the index range for `RPT_MONTH_YEAR`, in every record? if we're going to continue using this pattern for other fields, we may want to look at some sort of abstraction or modification to the `RowSchema`

Reply:
> is `[2:8]` always going to be the index range for `RPT_MONTH_YEAR`, in every record? 

yep. for files that have this data element, its always the first data element after record type.